great-idea-website/css/index.css

```diff
@@ -63,4 +63,125 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+p {
+	line-height: 1.15;
+	padding-right: 5px;
+}
+
+/* Your code starts here! */
+.container {
+	margin: 0 auto;
+	width: 1024px;
+	padding: 0 75px;
+}
+
+/* NAVIGATION */
```

Comment:
Good job commenting out the different sections so you can come back to it later and easily find things. It's a good habit to have early on